test/mongorito.test.js

```diff
@@ -31,33 +31,8 @@ const postFixture = require('./fixtures/post');
 */
 
 describe ('Mongorito', function () {
-  it ('expose mongoskin module properties', function () {
```

Comment:
It's not a good practice to just remove a test to make a whole test suite pass. This was here for a reason. People need access to various properties exposed by underlying driver. Most common one is ObjectId.